src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1808,6 +1814,9 @@ private void OnOrganelleToPlaceSelected(string organelle)
         if (ActiveActionName == organelle)
             return;
 
+        var organelleDefinition = SimulationParameters.Instance.GetOrganelleType(organelle);
```

Comment:
I found one bug regarding this, if I select a unique organelle after turning on symmetry, then if I save and load the game I end up in a state where the unique organelle is selected to be placed *but* the symmetry button is enabled. I think in the component `Init` around the other post save load actions it'd be good to re-apply the symmetry state. Probably better to write a small helper method that gets called here and there to ensure consistent applying the GUI state.

Reply:
Actually looks like "cytoplasm" selection is done in `UpdateGUIAfterLoadingSpecies` so I think this second call to that new helper method I asked you to make should go there instead as the logical place for it to be.